dss/storage/checkout.py

```diff
@@ -76,6 +76,26 @@ def start_bundle_checkout(
     return execution_id
 
 
+def start_file_checkout(
+        blob_key,
+        replica: Replica,
+        dst_bucket: typing.Optional[str]=None,
+) -> str:
+    """
+    Starts a file checkout.
+
+    :param blob_key: The key of the blob that contains the file.
+    :param replica: The replica to execute the checkout in.
+    :param dst_bucket: If provided, check out to this bucket.  If not provided, check out to the default checkout bucket
```

Comment:
Could this be a user-provided bucket?

Reply:
This API supports a user-provided bucket.  There is no endpoint to accomplish that, however.